examples/tensor_flow/readme.md

```diff
@@ -0,0 +1,252 @@
+
+`pachctl list-job`
+
+and 
+
+`pachctl list-commit GoT_generate`
+
+The first command will show you the current status of the jobs. In our case, we expect the training pipeline to run for much longer than the generate step.
+
+#### Results:
+
+By default, we've set the size of the model to train to `test`. You can see this on each of the entrypoints specified in the `pipeline.json` file. For this size model, the training pipeline should run for a couple minutes, and the generate step much quicker than that. Let's take a look at the output.
+
+Once `pachctl list-commit GoT_generate` shows a single commit, we can take a look at the output. To do so, you can run:
+
+```
+pachctl get-file GoT_generate {the commit id from the above command} new_script.txt
+```
+
+And you should see some text!
+
+Keep in mind, the model we just trained was very simplistic. Doing the 'test' model suffices as a proof of concept that we can train and use a neural net using Tensor Flow on Pachyderm. That said, the test model is dumb, and the output won't be that readable.
+
+Actually, you can see how 'dumb' the model is. If you [read through the Tensor Flow example](https://www.tensorflow.org/versions/r0.8/tutorials/recurrent/index.html#run-the-code) they describe how 'perplexity' is used to measure how good this model will perform. Let's look at the perplexity of your model.
+
+```
+pachctl list-commit GoT_trian
+pachctl get-file GoT_train {commit from command above} model_results
+```
+
+Remember, this file is the stdout from the python script while its training the model. It outputs the 'perplexity' at standard intervals on the training set, as well as outputs the 'perplexity' for the validation and test sets. You should see a perplexity of about 600 or less.
+
+That's not great. As a next step, you can improve that measure, and the readability of the output script!
+
+#### Next Iteration
+
+As referenced in the [Tensor Flow example](https://www.tensorflow.org/versions/r0.8/tutorials/recurrent/index.html#run-the-code), a 'perplexity' of less than 100 starts to get pretty good / readable. Running the 'small' model on the GoT data set should produce a perplexity lower than 100.
+
+To do this, you'll need to tear down this pipeline and re-create it. Specifically:
+
+1) Delete the existing data / pipeline by running:
+
+```
+pachctl delete-all
+```
+
+2) Change the entrypoint commands in `pipeline.json` under the transformation property to use the `small` model not the `test` model. Save your changes.
+
+3) Re-initialize the data / compile the image / create the pipeline
+
+Now you can do this in one step:
+
+```
+make all
+```
+
+The small model runs in about an hour. Once its complete, you can look at the output again. This time, the perplexity should be south of 100 and the output script should be semi-readable.
+
+---
+
+##3 Example Output
+
```

Comment:
Remove "3". Also, add a bunch of description describing how freaking cool this is. This model didn't know english, or names or the difference between dialogue and expositions, but it clearly figure'd it out most of the time.